From 2e6f906c4930cb6621170f6592889b48c57eca58 Mon Sep 17 00:00:00 2001
From: Huang Chaofan <chaofan.huang@nxp.com>
Date: Thu, 8 Nov 2018 10:52:08 +0800
Subject: [PATCH 5063/5242] MLK-20274 VPU: Change request for copyright in vpu
 header

commit  dd7f6685aa443d3bf40e5b15e506d7b251f9646f from
https://source.codeaurora.org/external/imx/linux-imx.git

Remove several lines in the file VPU_regdef.h to avoid license conflict

Signed-off-by: Huang Chaofan <chaofan.huang@nxp.com>
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 drivers/mxc/vpu-malone/VPU_regdef.h |    3 ---
 1 file changed, 3 deletions(-)

diff --git a/drivers/mxc/vpu-malone/VPU_regdef.h b/drivers/mxc/vpu-malone/VPU_regdef.h
index 4753a90..597cc13 100755
--- a/drivers/mxc/vpu-malone/VPU_regdef.h
+++ b/drivers/mxc/vpu-malone/VPU_regdef.h
@@ -2,9 +2,6 @@
  * Copyright 2017 NXP
  */
 
-/* The following programs are the sole property of NXP Semiconductor Inc.,
- * and contain its proprietary and confidential information. */
-
 /*
  * The code contained herein is licensed under the GNU General Public
  * License. You may obtain a copy of the GNU General Public License
-- 
1.7.9.5

